include/rocksdb/options.h

```diff
@@ -962,6 +962,14 @@ struct DBOptions {
   // Default: 1000
   size_t keep_log_file_num;
 
+  // Recycle log files.
+  // If true, we will reuse previously written log files for new logs,
+  // overwriting the old data.  This is more efficient because the blocks
+  // are already allocated and fdatasync does not need to update the inode
+  // after each write.
+  // Default: false
```

Comment:
I think it depends on the other logging parameter (max_write_buffer_number). In our case, I plan to set that to something largish (16?) along with min_write_buffer_to_merge=6 or something so that I can minimize the number of short-lived records that get rewritten.